manifest.json

```diff
@@ -19,7 +19,7 @@
     },
     "services": [
         "nginx",
-        "php5-fpm",
+        "php7.0-fpm",
```

Comment:
Shouldn't that be "php-73.0-fpm"?

Reply:
The CI is not happy with the service set to `php7.3-fpm` and fails ... I'm guessing something needs to be updated somewhere to fix this..